woocommerce/wc-admin
# Removed @woocommerce/components/card from OBW

Fixes #6166 

This PR removes the usage of "@woocommerce/components/card" from the step "Business Details" in the OBW.

I also looked for other uses of `@woocommerce/components/card` but I couldn't find any.

### Accessibility




### Screenshots
![screenshot-one wordpress test-2021 02 18-16_52_20](https://user-images.githubusercontent.com/1314156/108413429-c9274c00-7209-11eb-8f25-937e0f40fbb5.png)


### Detailed test instructions:

- Checkout this branch (`git checkout fix/6166`)
- Go to the `Business Details` step on the OBW (URL `/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard&step=business-features`).
- Verify everything looks good and there aren't any console errors.
- Do some smoke testing of the OBW.
